pkg/subctl/cmd/cloud/cleanup/gcp.go

```diff
@@ -20,38 +20,31 @@ package cleanup
 
 import (
 	"github.com/spf13/cobra"
-	"github.com/submariner-io/admiral/pkg/reporter"
-	"github.com/submariner-io/cloud-prepare/pkg/api"
 	"github.com/submariner-io/submariner-operator/internal/cli"
 	"github.com/submariner-io/submariner-operator/internal/exit"
+	"github.com/submariner-io/submariner-operator/internal/restconfig"
+	"github.com/submariner-io/submariner-operator/pkg/cloud/cleanup"
+	gcpcloud "github.com/submariner-io/submariner-operator/pkg/cloud/gcp"
 	"github.com/submariner-io/submariner-operator/pkg/subctl/cmd/cloud/gcp"
 )
 
+var gcpConfig gcpcloud.Config
```

Comment:
This is unused and can be removed.

Reply:
Unused yes, but `AddGCPFlags()` needs it as it is needed for `prepare`. I made the variable local instead of global.